README.md

```diff
@@ -4,10 +4,39 @@ DotNetTools
 [![Travis build status](https://img.shields.io/travis/aspnet/DotNetTools.svg?label=travis-ci&branch=dev&style=flat-square)](https://travis-ci.org/aspnet/DotNetTools/branches)
 [![AppVeyor build status](https://img.shields.io/appveyor/ci/aspnetci/DotNetTools/dev.svg?label=appveyor&style=flat-square)](https://ci.appveyor.com/project/aspnetci/DotNetTools/branch/dev)
 
-The project contains command-line tools for the .NET Core CLI.
+This project is part of ASP.NET Core. You can find samples, documentation and getting started instructions for ASP.NET Core at the [Home](https://github.com/aspnet/home) repo.
 
- - [dotnet-watch](src/Microsoft.DotNet.Watcher.Tools/)
- - [dotnet-user-secrets](src/Microsoft.Extensions.SecretManager.Tools/)
- - [dotnet-sql-cache](src/Microsoft.Extensions.Caching.SqlConfig.Tools/)
+## Projects
 
-This project is part of ASP.NET Core. You can find samples, documentation and getting started instructions for ASP.NET Core at the [Home](https://github.com/aspnet/home) repo.
+The repository contains command-line tools for the .NET Core CLI.
+
+ - [dotnet-watch](src/Microsoft.DotNet.Watcher.Tools/) (Microsoft.DotNet.Watcher.Tools)
```

Comment:
FYI there are README's already for each of these tools that explain what each does.